db/schema.rb

```diff
@@ -81,6 +98,18 @@
     t.index ["project_id"], name: "index_featured_circuits_on_project_id"
   end
 
+  create_table "grades", force: :cascade do |t|
+    t.string "grade"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.bigint "project_id"
+    t.bigint "user_id"
+    t.bigint "assignment_id"
+    t.index ["assignment_id"], name: "index_grades_on_assignment_id"
+    t.index ["project_id"], name: "index_grades_on_project_id"
+    t.index ["user_id"], name: "index_grades_on_user_id"
+  end
```

Comment:
Why is grades table change here?

Reply:
Once the previous PR gets merged, I'll rebase this branch and this should go away then.